### PR TITLE
Add page: Community

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

File: _pages/community.html [hunk 1]
```diff
@@ -1,187 +1,285 @@
...
+      <div class="link-item">
+        <a class="link-item__main" href="https://medium.com/tag/crystal-lang">
+          {% include icons/medium.svg %}
+          Crystal on Medium</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://lbarasti.com/tags/crystal/">
+          {% include icons/newsmode.svg %}
+          Lorenzo Barasti</a>
+      </div>
```
Comment (beta-ziliani) [inline _pages/community.html:253]: I think this belongs to the External resources page in the documentation, but the call is tight.

Comment (beta-ziliani) [reply _pages/community.html:253]: In any case, the name isn't very telling :-D

File: _pages/community.html [hunk 2]
```diff
@@ -1,187 +1,285 @@
...
+      <div class="link-item">
+        <a class="link-item__main" href="https://manas.tech/blog/">
+          {% include icons/manas.svg %}
+          Manas.Tech Blog</a>
+      </div>
```
Comment (beta-ziliani) [inline _pages/community.html:258]: This is not Crystal specific

File: _pages/community.html [hunk 3]
```diff
@@ -1,187 +1,285 @@
...
+    </div>
+  </div>
+</section>
+
+{% comment %}
```
Comment (beta-ziliani) [inline _pages/community.html:263]: What is this?

**Review by beta-ziliani — COMMENTED** [review]
sorry for the bombing!

File: _pages/community/governance.md
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _pages/community/governance.md:1]: Well I'm standardizing the format of all source files using `markdownlint` (it's part of the pre-commit hook installed in devenv) and this is what it produces. I think it's good. And it improves readability (and editability).
I'm happy to pull out these changes if you'd rather not have them here.

File: _pages/community.html [hunk 4]
```diff
@@ -1,187 +1,285 @@
...
-  link_text="crystal-lang.org/media"
-  url="/media"
-  icon="palette"
-%}
-
-</div>
+<section id="channels">
+  <div class="topics">
+    <div id="discuss" class="link-items">
+      <h3>Discuss</h3>
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _pages/community.html:9]: Maybe. But I think that would actually be quite complex and not improve much. Using raw HTML offers a lot of flexibility.

File: _pages/community.html [hunk 5]
```diff
@@ -1,187 +1,285 @@
...
+    <p>
+      <a href="/team">
+        <img src="https://opencollective.com/crystal-lang/contributors.svg?width=960">
+      </a>
+    </p>
+
+    <p>
+      The Crystal community is lead by the Core Team and the Steering Council,
+      as defined in the <a href="/community/governance">Governance document</a>.
+    </p>
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _pages/community.html:137]: My intention was to insert a reference to the governance doc. But I wasn't too convinced about this either.

File: _pages/community.html [hunk 6]
```diff
@@ -1,187 +1,285 @@
...
+  </div>
+</section>
+
+<section id="ecosystem">
+  <a id="shards">
+  <h2>Crystal Ecosystem</h2>
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _pages/community.html:143]: The spacing was an error. I fixed that.

About the distance to the parent, please check out how it looks after the removal of the text.

File: _pages/community.html [hunk 7]
```diff
@@ -1,187 +1,285 @@
...
+          {% include icons/podcasts.svg %}
+          Chicago Crystal Podcast
+        </a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://www.youtube.com/channel/UCI1RvHPG6S9mw4eRoJfH2kA">
+          {% include icons/subscriptions.svg %}
+          Chicago Crystal Videos</a>
+      </div>
+      <div class="link-item">
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _pages/community.html:229]: See the TODO below.

File: _pages/community.html (hunk 2 above)
Comment (straight-shoota) [reply _pages/community.html:258]: Not necessarily, yes. Still a blog we can recommend, thogh 🤷 I don't mind removing it here, though.

File: _pages/community.html (hunk 3 above)
Comment (straight-shoota) [reply _pages/community.html:263]: The events section requires the events page which is still pending. I still did some basic work on it and didn't want to lose it, so it's checked in but commented with a TODO for picking up later in combination with the events page.

File: _pages/community.html (hunk 1 above)
Comment (beta-ziliani) [reply _pages/community.html:244]: Not really, this is a personal blog post. The others are for anybody that wants to `tag:crystal`

Comment (beta-ziliani) [reply _pages/community.html:244]: Perhaps I'm just missing what's the line to put things here

File: _pages/community.html (hunk 2 above)
Comment (beta-ziliani) [reply _pages/community.html:249]: https://manas.tech/blog/tagged/crystal/ maybe?

Comment (beta-ziliani) [reply _pages/community.html:249]: and it might be stupid, but we need to link the `/blog` page too!

File: _pages/community.html (hunk 1 above)
Comment (straight-shoota) [reply _pages/community.html:244]: Sure, dev.to and Medium are aggregates. But in the end they're just a collection of personal blogs. The only difference is that they're accessed via a shared provider vs. others are independent.

Comment (beta-ziliani) [reply _pages/community.html:249]: OK, but I would at least link personal blog of people that regularly post about Crystal. Lorenzo might still make the cut (although the last one is from 3yrs ago...), and Manas definitively doesn't fit.

Comment (beta-ziliani) [reply _pages/community.html:249]: I'd say to proceed conservatively, and then we can add as we see fit

**Comment by beta-ziliani** [comment]
the failures are expected?

**Comment by straight-shoota** [comment]
Yeah the newsletter URLs don't work yet because #476 is not in the branch. It's okay, though. I'll merge master into the feature branch when this is merged.

File: _pages/community.html (hunk 1 above)
Comment (straight-shoota) [reply _pages/community.html:249]: IMO regular posting should not be a necessary critererion. I'm more interested in high quality content that is (still) relevant. And I think both Lorenzo as well as Manas match that. But we're really diverting from the main purpose of implementing the design.
We can simply drop those two links, or the entire blogs section and then come back with a separate discussion about the contents.

Comment (beta-ziliani) [reply _pages/community.html:249]: sounds good, let's leave it for now

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**